### ISSUE TITLE
remote write to influx not working with prometheus 2.2.0

### ISSUE THREAD

**Issue by adityanmishra** [prometheus/prometheus#3965]
**What did you do?**
I am trying to use influx db for remote storage to keep data for long without impacting prometheus.
Used prometheus docker image for 2.2.0 and remote write to influx is stopped. I don't see any error logs also.

I then used 2.1.0 remote write starts working.  I am not sure what am i doing wrong. Any help will be appreciated.

**What did you expect to see?**
With 2.2.0 data should be posted to Influxdb

**What did you see instead? Under which circumstances?**
Nothing no error message

**Environment**
rhel 7

* System information:

	insert output of `uname -srm` here
Linux 3.10.0-693.5.2.el7.x86_64 x86_64

* Prometheus version:
2.2.0
	insert output of `prometheus --version` here

* Alertmanager version:

	insert output of `alertmanager --version` here (if relevant to the issue)

* Prometheus configuration file:
Prometheus.yml
-------------------
global:
  scrape_interval: 5s

rule_files:
  - '/etc/prometheus/rules/*.yml'

remote_write:
  - url: "http://*.*.*.*:8086/api/v1/prom/write?u=prom&p=****&db=prometheus"

scrape_configs:
- job_name: etcd-etcd-cluster-tlp
  scrape_interval: 30s
  scrape_timeout: 10s
  metrics_path: /metrics
  scheme: http
  static_configs:
  - targets:
    - *.*.*.*:2379
    - *.*.*.*:2379
    - *.*.*.*:2379

docker-compose.yaml
---------------------
version: '2'
services:
  prometheus:
    image: prom/prometheus:v2.2.0
    container_name: prometheus
    user: root
    volumes:
      - /*/*/prom/prometheus.yml:/etc/prometheus/prometheus.yml
    command:
      - '--config.file=/etc/prometheus/prometheus.yml'
      - '--web.listen-address=:8080'
      - '--web.enable-lifecycle'
      - '--storage.tsdb.retention=5h'
      - '--log.level=debug'
```
insert configuration here
```

* Alertmanager configuration file:
```
insert configuration here (if relevant to the issue)
```

* Logs:
Creating network "prom_default" with the default driver
Creating prometheus
Attaching to prometheus
prometheus    | level=info ts=2018-03-14T15:48:47.836048906Z caller=main.go:220 msg="Starting Prometheus" version="(version=2.2.0, branch=HEAD, revision=f63e7db4cbdb616337ca877b306b9b96f7f4e381)"
prometheus    | level=info ts=2018-03-14T15:48:47.836281701Z caller=main.go:221 build_context="(go=go1.10, user=root@52af9f66ce71, date=20180308-16:40:42)"
prometheus    | level=info ts=2018-03-14T15:48:47.836370228Z caller=main.go:222 host_details="(Linux 3.10.0-693.5.2.el7.x86_64 #1 SMP Fri Oct 13 10:46:25 EDT 2017 x86_64 1140a5848890 (none))"
prometheus    | level=info ts=2018-03-14T15:48:47.836433627Z caller=main.go:223 fd_limits="(soft=65536, hard=65536)"
prometheus    | level=info ts=2018-03-14T15:48:47.857721007Z caller=main.go:504 msg="Starting TSDB ..."
prometheus    | level=info ts=2018-03-14T15:48:47.873508728Z caller=web.go:382 component=web msg="Start listening for connections" address=:8080
prometheus    | level=info ts=2018-03-14T15:48:47.889321876Z caller=main.go:514 msg="TSDB started"
prometheus    | level=info ts=2018-03-14T15:48:47.889426487Z caller=main.go:588 msg="Loading configuration file" filename=/etc/prometheus/prometheus.yml

```
insert Prometheus and Alertmanager logs relevant to the issue here
```

**Comment by Gouthamve** [comment]
Looks like a dupe of https://github.com/prometheus/prometheus/issues/3809 and the fix has been released in 2.2.1 here: https://github.com/prometheus/prometheus/releases/tag/v2.2.1

Closing this, but if you still see this in the latest release, please feel free to reopen.

**Comment by adityanmishra** [comment]
It is fixed with 2.2.1. Thanks a lot
